content/en/blog/2020/deploy-wasm-declarative/index.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Declarative WASM deployment for Istio
+subtitle: Deploy extensions to Envoy in Istio using a declarative model to align with GitOps workflows
+description: Configuring WASM extensions for Envoy and Istio declaratively.
+publishdate: 2020-03-16
+attribution: "Christian Posta (Solo.io)"
+keywords: [wasm,extensibility,alpha,operator]
+---
+
+As outlined in the [Istio 2020 Tradewinds blog](/blog/2020/tradewinds-2020/) and more recently [announced with Istio 1.5](/news/releases/1.5.x/announcing-1.5/), WebAssembly (Wasm) is now an (alpha) option for extending the functionality of the Istio service proxy (Envoy proxy). With Wasm, users can build support for new protocols, custom metrics, loggers, and other filters. Working closely with Google, we in the community ([Solo.io](https://solo.io)) have focused on the user experience of building, socializing, and deploying Wasm extensions to Istio. We've announced [WebAssembly Hub](https://webassemblyhub.io) and [associated tooling](https://docs.solo.io/web-assembly-hub/latest/installation/) to build a "docker-like" experience for working with Wasm. 
+
+
+## Background
+
+With the WebAssembly Hub tooling, we can use the `wasme` CLI to easily bootstrap a Wasm project for Envoy, push it to a repository, and then pull/deploy it to Istio. For example, to deploy a Wasm extension to Istio with `wasme` we can run the following:
+
+```
```

Comment:
Replace the ``` sections with:
```
{{< text bash >}}
$ wasme deploy istio webassemblyhub.io/ceposta/demo-add-header:v0.2 \
...
{{< /text >}}
```
 Note that you also need to add $ at the beginning of the first line of a bash text section.

Reply:
Thanks @frankbu I've updated based on your suggestions here